src/server/server_family_test.cc

```diff
@@ -200,4 +200,52 @@ TEST_F(ServerFamilyTest, ClientPause) {
   EXPECT_GT((absl::Now() - start), absl::Milliseconds(50));
 }
 
+TEST_F(ServerFamilyTest, ClientTracking) {
```

Comment:
I would create different unit test for different test cases.
I think that it is great that we have this in unit tests with mocking but we must add pytest to make sure this command works well to check the functionality without the mocking

Reply:
that's fine with me too. Currently i see different styles mixed in our code: sometimes all cases went into one function (e.g. ClientPause here, and many other redis commands' tests), sometimes separated smaller cases (e.g. SlowLog here)